packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -96,29 +156,21 @@ VTL syntax:
 | escapeHtml(String html)                                       | Escapes the characters in the provided string using HTML entities                                                             |
 | escapeHtmlAttr(String html)                                   | Encodes data for use in HTML attributes                                                                           |
 
-##### Examples of user attributes
-
-###### Registration: Email verification with customer's preferred language
-Trigger an email in your end-users' preferred language by using the conditional logic and calling the `preferredLanguage` attribute.
-
-###### Activation department specific email
-If a customer wants to specify a department such as Engineering in an Activation Email, they are able to call the user attribute `department`.
-
-#### Access to org attributes
+### Use org attributes
 
 You can also reference these org-level attributes, such as:
 
 * `$org.name`
 * `$org.locale`
 * `$org.subDomain`
 
-#### Support for conditional logic
+### Support for conditional logic
 
 All conditional logic that is supported by the Velocity Templating Engine, such as `if`, `elseif`, or `else` constructs and `foreach` loops, is available for you to use in your templates. See the [Velocity documentation](http://velocity.apache.org/engine/1.7/user-guide.html).
 
 <ApiLifecycle access="ie" /><br>
```

Comment:
I'm assuming this is an OIE-only feature? I'd expect to see the Identity Engine badge at the top of the section, not the bottom.

Reply:
It also doesn't need the `<br>` — this just serves to add unnecessary extra space below the badge.